core/src/main/java/org/keycloak/representations/account/ClientRepresentation.java

```diff
@@ -1,11 +1,20 @@
 package org.keycloak.representations.account;
 
+import java.util.List;
+
 /**
  * Created by st on 29/03/17.
  */
 public class ClientRepresentation {
     private String clientId;
     private String clientName;
+    private String description;
+    private boolean internal;
+    private boolean inUse;
+    private String url;
```

Comment:
Rename to baseUrl to make it clear that it is the base-url of the client. There may be other URLs in the future.